libnopegl/src/pgcraft.c

```diff
@@ -1222,7 +1233,8 @@ static int32_t get_texture_index(const struct pgcraft *s, const char *name)
     const struct pipeline_resource_desc *pipeline_texture_descs = ngli_darray_data(&s->filtered_pipeline_info.desc.textures);
     for (int32_t i = 0; i < (int32_t)ngli_darray_count(&s->filtered_pipeline_info.desc.textures); i++) {
         const struct pipeline_resource_desc *pipeline_texture_desc = &pipeline_texture_descs[i];
-        if (!strcmp(pipeline_texture_desc->name, name))
+        const char *texture_name = ngli_pgcraft_get_symbol_name(s, pipeline_texture_desc->id);
+        if (!strcmp(texture_name, name))
```

Comment:
Yet again you can call `get_symbol_name` only once out of the loop

Reply:
Same response as for the other case.